### ISSUE TITLE
Is it possible to replicate whatsapp call receiving behaviour

### ISSUE THREAD

**Issue by saaimbiz** [rafaelsetragni/awesome_notifications#284]
Using this plugin I was able to show time sensitive notification with the default ringtone.

I used 
```
defaultRingtoneType: DefaultRingtoneType.Ringtone,
importance: NotificationImportance.High,
locked: true
```

Here is a preview
![image](https://user-images.githubusercontent.com/81901191/132956626-8bd069f4-6b0c-4c60-b56b-867a2125d11d.png)

But I couldn't complete following two

1. Keep this notification for a longer period of time ( until the ringtone stops or for a fixed time such as 20 sec )
2. When the screen is off ( in lock ), wake up screen

Please guide me in the right direction

**Comment by rafaelsetragni** [comment]
Yes! This is the next feature that ill release. But for now im working with receiving background notifications. Only after that (0.0.7 version) ill be able to develope this new feature.

**Comment by maverick-2013** [comment]
@rafaelsetragni This feature is related to Notification.CATEGORY_CALL, right? How long will it take to implement?

**Comment by rafaelsetragni** [comment]
I need to release the 0.0.7 version and finish the companion plugin to receive push notifications from Firebase. I gonna need 2-3 months to get at this point.

**Comment by rafaelsetragni** [comment]
Its already available at version 0.6.14 with two versions: Messaging and Messaging Group.

**Comment by vaartatechnologies** [comment]
Hi there,
`    any development on

    await AwesomeNotifications().createNotification(
        wakeUpScreen: true,
        category: NotificationCategory.Call,
    ),`
I have been waiting since your last comment on `Sep 21, 2021`.

Thanks in advance for the plugin.

**Comment by rafaelsetragni** [comment]
Hi! Sorry for taking so long to respond to your issue on GitHub. I was focused on releasing the new 0.7.0 version of `awesome_notifications` and `awesome_notifications_fcm` and i didn't had time enough to do both. But now I can answer all your questions.

So now i'm asking you to recreate this topic using the new issue template. There's a lot of missing informations that i need to understand your problem, and all the instructions are already in the new issue template. Also, remember to check beforehand if your issue was posted by another user.

So I will automatically close all previous issues so far.
Sorry for the inconvenience and i will be waiting for your new issue request.

Thank you so much for your support!
